dotCMS/src/main/java/com/dotcms/rest/api/v1/index/ESIndexResource.java

```diff
@@ -484,15 +484,18 @@ public Response getActive(@Context HttpServletRequest httpServletRequest, @Conte
     @Path("/docscount/{params:.*}")
     @Produces("text/plain")
     public Response getDocumentCount(@Context HttpServletRequest httpServletRequest, @Context final HttpServletResponse httpServletResponse, @PathParam("params") String params) {
+
+        InitDataObject init= null;
         try {
-            InitDataObject init=auth(params,httpServletRequest, httpServletResponse);
+            init = auth(params,httpServletRequest, httpServletResponse);
 
             //Creating an utility response object
             ResourceResponse responseResource = new ResourceResponse( init.getParamsMap() );
 
             String indexName = this.indexHelper.getIndexNameOrAlias(init.getParamsMap(),"index","alias",this.indexAPI);
             return responseResource.response( Long.toString( indexDocumentCount( indexName ) ) );
-        } catch (Exception e) {
+
+        } catch (DotSecurityException | DotDataException e) {
```

Comment:
I think we should remove this catch also and let the Mappers catch these Exceptions

Reply:
The problem with doing this is that it would change the method signature. That's why I left those 2 exceptions